### PR TITLE
Fix #1004: add connector for groovemp3.com

### LINKED ISSUE #1004: Add support for groovemp3.com (new version of Grooveshark)

### PR DESCRIPTION
**Pull request by ex47**
#1004

Currently ignores other artists if there are more than one.

### CODE REVIEW

File: core/connectors.js [hunk 1]
```diff
@@ -1020,6 +1020,13 @@ define(function() {
 			matches: ['*://*radiobonton.cz/*'],
 			js: ['connectors/v2/danceradiocz.js'],
 			version: 2
+		},
+
+		{
+			label: 'GrooveMP3',
+			matches: ['*://groovemp3.com/*'],
```
Comment (Paszt) [inline core/connectors.js]: _www_ subdomain is also valid, please include that: http://www.groovemp3.com/  

    matches: ['*://groovemp3.com/*', '*://www.groovemp3.com/*'],

**Comment by alexesprit** [comment]
Just noticed that groovemp3 has the same layout as youtubify.

**Comment by inverse** [comment]
Do you think they should just use the same connector?

**Comment by alexesprit** [comment]
I think there's no reason to duplicate the same thing. But in this case the connector should have the common name for both websites (e.g. used UI template name).

**Comment by alexesprit** [comment]
Looks like it's a [BeMusic music streaming engine](https://codecanyon.net/item/bemusic-music-streaming-engine/13616699). So, I think the `youtubify` connector should be renamed to `bemusic` and should be used for both websites.

**Comment by ex47** [comment]
so what do we do?

**Comment by alexesprit** [comment]
Well, I suggested an option already. :)

Or you can resolve conflicts and leave the connector as it is, but it should be fixed (see #1150 for details),

**Comment by alexesprit** [comment]
Impelemented in 6da7be655ebcecf0b7cf7c799a70ad9e97253b37.

**Pull request closed without merging**